File: src/easy/age_to_days.zig
const expect = @import("std").testing.expect;

pub fn ageToDays(age: u16) u16 {
    return age * 365;
}

test "Age on earth from days 3_650 to years 10" {
    try expect(ageToDays(10) == 3650);
}
test "Age on earth from days 0 to years 0" {
    try expect(ageToDays(0) == 0);
}
test "Age on earth from days 26+645 to years 73" {
    try expect(ageToDays(73) == 26645);
}

File: src/easy/age_to_days.rs
pub fn age_to_days(age: u16) -> u16 {
    age * 365
}
#[cfg(test)]
mod tests {
    use super::age_to_days;
    #[test]
    fn test_age_to_days() {
        assert_eq!(age_to_days(10), 3650);
        assert_eq!(age_to_days(0), 0);
        assert_eq!(age_to_days(73), 26645);
    }
}
